src/main/java/core/basesyntax/SquareOfFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface SquareOfFigure {
+    double getArea();
```

Comment:
```suggestion
public interface AreaCalculator {
    double getArea();
```

as we can use this interface not only for figures